modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -886,7 +886,17 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       SendShareType.SShare,
       userSShare
     );
-    return await getTxRequest(this.bitgo, this.wallet.id(), txRequestObj.txRequestId);
+    try {
+      return await getTxRequest(this.bitgo, this.wallet.id(), txRequestObj.txRequestId);
+    } catch (error) {
+      if (error.status === 502) {
+        throw new Error(
+          'Transaction broadcasted successfully, but BitGo failed to update the transaction status ' + error
```

Comment:
We seem to be doing a get and not update

Reply:
+1

Reply:
fixed it.

Reply:
have added couple of tests.